### DISCUSSION TITLE
how to turn off auto-update with git

### DISCUSSION THREAD

**Discussion by cjps-br** [vladmandic/sdnext#314] [Q&A]
how can i disable auto update from automatic? I noticed that whenever there are commits I get the update even without having to do git pull. I'd like to be in control of this, updating when I feel I need to.

**Comment by vladmandic** [comment]
```
  --upgrade                            Upgrade main repository to latest version, default: False
  --noupdate                           Skip update of extensions and submodules, default: False
  --skip-requirements                  Skips checking and installing requirements, default: False
  --skip-extensions                    Skips running individual extension installers, default: False
  --skip-git                           Skips running all GIT operations, default: False
```

**Reply by cjps-br** [reply]
Thanks. I was using commit ee3cea5 and everything was working on generation. But today, when starting to do a job, I've been getting only all black images. The environment was automatically updated to commit 0282832f so I want to go back to commit ee3cea5 and then stay there until I finish the work I'm doing. Then I can search why I'm not getting images in this new commit

**Reply by vladmandic** [reply]
sure. but black images scare me and i've heard from other random people about NaNs appearing more often - and there are no core changes that should result in that.

**Reply by cjps-br** [reply]
I saw that the settings under CUDA moved from the arguments line to the settings tab. I use the following command arguments: set COMMANDLINE_ARGS=--xformers --precision full --no-half --medvram

I use this line of arguments from this web-ui SD1111.


this works and generates images in commit b298b002.

After I finish my main work, I will test which one effectively generates black images when not present and try to put the corresponding one in the new system settings.

**Reply by fayne10** [reply]
Sorry which files do i add the " --" commands in?

**Reply by 779575267** [reply]
same question！！！launch.py: error: unrecognized arguments: --skip-requirements

**Comment by cjps-br** [comment]
with the following configuration in the settings, I was able to have generated images again and no more black images

    "precision": "Full",
    "cuda_dtype": "FP32",
    "no_half": true,
    "no_half_vae": true,
    "cuda_allow_tf32": true,
    "cuda_allow_tf16_reduced": true,
    "cuda_compile": false,
    "cuda_compile_mode": "none"


![image](https://user-images.githubusercontent.com/75840971/233517768-a63cad02-faa1-446a-ab37-a6aacd661af8.png)

**Comment by 779575267** [comment]
launch.py: error: unrecognized arguments: --skip-git

**Reply by tkiikt** [reply]
#275
